src/routes/recipe.js

```diff
@@ -1,17 +1,62 @@
 import React from 'react';
 import { useParams } from "react-router-dom";
 import { Link } from "react-router-dom";
+import { getRecipe } from '../recipes';
+
+function IngredientItem(props) {
+  const listItems = props.ingredients.map(ingredient => 
```

Comment:
Minor: Is there any reason not to just return `props.ingredients.map(...)` instead of saving to a variable and then returning the variable?